qoe-metrics-dashboard/src/app/pages/overview/Overview.tsx

```diff
@@ -112,7 +113,7 @@ function Overview() {
 
     return (
         <div className="page-wrapper">
-            <ReloadButton action={onReload} />
+            <ReloadButton action={onReload} topic={ESseTopic.CONSUMPTION}/>
```

Comment:
this should be listening on the QoE Reports, not the consumpition reports right? 

We still need a different overview for the Qoe and the consumption reports?

Reply:
Currently in the frontend we have just metrics reports therefor it should be 
```suggestion
            <ReloadButton action={onReload} topic=topic={ESseTopic.METRICS}/>
```

If you mean we should rename metrics to QoE we can do this however then i would *not* do this in that PR but in a refactor PR. Till this date we often used metrics as route or in docs.